graphene/src/boolean_ops.rs

```diff
@@ -453,12 +473,77 @@ pub fn subdivide_path_seg(p: &PathSeg, t_values: &mut [f64]) -> Vec<Option<PathS
 	sub_segments
 }
 
+/// * shapes.len() > 1
```

Comment:
This doc comment is meant to describe the functionality of this function, but the comment doesn't really make any sense to me.